crypto/x509/x509_vfy.c

```diff
@@ -633,6 +633,28 @@ static int check_custom_critical_extensions(X509_STORE_CTX *ctx, X509 *x) {
   return 1;
 }
 
+static int check_curve(X509 *x) {
```

Comment:
It's not clear what the return value means. A name like `check_curve_decoded_by_name` would be more clear.

Reply:
I was keeping the name to match the OpenSSL corresponding function for simplicity in comparing implementations.